audio-buffer-source-node/loop/script.js

```diff
@@ -0,0 +1,63 @@
+const audioCtx = new AudioContext();
```

Comment:
This raises an error in Chrome (only) that doesn't allow the creation of `AudioContext` object before a user gesture on the page. (Security requirement: The _Autoplay policy in Chrome_ https://goo.gl/7K7WLu ).
![Capture d’écran 2023-11-16 à 10 02 59](https://github.com/mdn/webaudio-examples/assets/1466293/17946eff-600b-43a6-835c-ba0a2946ca23)

The example does work, but we should avoid this. (I had to workaround around this the last time I updated some examples here)

Reply:
I've updated the script in the next comment (selecting the whole script.js file)

Question: do you want to update the other scripts (several have the same problem) or do you want to do ti yourself?